glfs.c

```diff
@@ -772,10 +766,17 @@ static int tcmu_glfs_reconfig(struct tcmu_device *dev,
 			/* Let the targetcli command return success */
 			ret = 0;
 		} else if (st.st_size != cfg->data.dev_size) {
+			/*
+			 * Currently we cannot update the size to kernel during
+			 * initialization, because it will be overrided by kernel
+			 * after it get the genl reply. Here will just allow it
+			 * to load the device and we will run gluster-block tool
+			 * to update the size.
+			 */
 			tcmu_dev_err(dev,
```

Comment:
@lxbsz , as I mentioned before, will warn level log is not enough?

Also please fix the code comment above.

Reply:
I just following the others, which is also as ERRORS:
```

Storage Object user/block0: Cannot set attribute dev_config: [Errno 95] Operation not supported, skipped
Storage Object user/block0: Cannot set attribute emulate_write_cache: [Errno 95] Operation not supported, skipped
Storage Object user/block0: Cannot set attribute dev_size: [Errno 22] Invalid argument, skipped
```

Reply:
@lxbsz I get you:
but look at the below logs:

```
2020-04-27 02:50:58.501 4589 [CRIT] main:1187: Starting...
2020-04-27 02:50:58.616 4589 [WARN] tcmu_glfs_open:558 glfs/block1: device size (1073741824) and backing file size (2147483648) not matching
2020-04-27 02:50:58.623 4589 [ERROR] tcmu_glfs_reconfig:714 glfs/block1: device size (1073741824) and backing file size (2147483648) not matching
```


They don't give any kind of impression that the device is loaded with the size same as backend anywhere. Rather they say we errored already.

Maybe some logs mentioning that we are ignoring the mismatch and continuing to load the storage with size same as backend file would be helpful IMHO.

Thanks!

Reply:
Yeah, sounds good.

Reply:
@pkalever  Updated it. Thanks